src/consensus/params.h

```diff
@@ -48,6 +48,7 @@ struct Params {
     int nBudgetProposalEstablishingTime; // in seconds
     int nSuperblockStartBlock;
     int nSuperblockCycle; // in blocks
+    int nMinGovernanceQuorum; // Min absolute vote count to trigger an action 
```

Comment:
I personally would prefer names here that are verbose enough, self explanatory and can be easily grouped under some pattern in case there are more e.g. in this specific case - `nGovernanceMinimumQuorum` or alike (which later can be nicely grouped with similar `nGovernanceWhatever` if any).